### ISSUE TITLE
Allow staff to register even if User obj already exists

### ISSUE THREAD

**Issue by aisapatino** [aisapatino/sjfnw#592]
Blocked by #505 - should be done after or as part of that

---

Sometimes new staff are set up with admin accounts and then want to use Project Central.

Current [registration](https://github.com/aisapatino/sjfnw/blob/master/sjfnw/fund/views.py#L282) tries to create a new User, so it will fail in that case.

Would be nice to have a special case for admin - they can register and get a Member/Membership created even if a User already exists.

Otherwise I need to add Member to the admin, which seems complicated for them.
